projects/commudle-admin/src/app/components/search-bar/search-bar.component.html

```diff
@@ -26,6 +26,15 @@
       <div *ngIf="showPlaceholder" class="placeholder-text d-flex justify-content-center align-items-center">
         Quickly search your favorite topics, labs by industry experts or find an event you missed
       </div>
+      <div *ngIf="showLoading" class="loading-text d-flex justify-content-center align-items-center">
+        <div class="spinner">
+          <div class="rect1"></div>
+          <div class="rect2"></div>
+          <div class="rect3"></div>
+          <div class="rect4"></div>
+          <div class="rect5"></div>
```

Comment:
Instead of different classes to each of these, we can use nth-child pseudo selector in css

Reply:
Done!